website/content/docs/provisioners/hcp-sbom.mdx

```diff
@@ -0,0 +1,130 @@
+---
+description: |
+  The `hcp-sbom` Packer provisioner downloads an SBOM file from the guest VM and
+  sends it to HCP Packer when the build is done.
+page_title: HCP SBOM - Provisioners
+---
+
+<BadgesHeader>
+  <PluginBadge type="official"/>
+</BadgesHeader>
+
+# HCP SBOM Provisioner
+
+Type: `hcp-sbom`
+
+The `hcp-sbom` Packer provisioner downloads an SBOM file from the guest machine
+and sends it to HCP Packer when the build is complete (only if the template is
+HCP-enabled). The SBOM file is automatically removed at the end of the process.
+If you want to retain a copy of the SBOM file, you can specify the
+`destination` option in the provisioner.
+
+Currently, we support `CycloneDX` and `SPDX` SBOM formats in `JSON`.
+
```

Comment:
TBA: Add more details about max number of files allowed to download, and if we are going to add the file name field!